profiles/Dijkstra-setup.yaml

```diff
@@ -308,43 +380,276 @@ hand_armor: *hand_armor # check the gear list to see whats going on here
+  price: 220000
+  surface: in cabinet
+  recipe: ring shirt
+  material: steel
+- noun: greaves
+  full_name: some steel ring greaves
+  price: 120000
+  surface: in cabinet
+  recipe: ring greaves
+  material: steel
+- noun: sipar
+  full_name: a steel round sipar
+  price: 200000
+  surface: in cabinet
+  recipe: round sipar
+  material: steel
+- noun: stick
+  full_name: parry stick
+  price: 120000
+  surface: on table
+  recipe: parry stick
+  material: steel
+- noun: hammer
+  full_name: throwing hammer
+  price: 120000
+  surface: on table
+  recipe: throwing hammer
+  material: steel
+- noun: spike
+  full_name: throwing spike
+  price: 100000
+  surface: on table
+  recipe: throwing spike
+  material: steel
+- noun: spear
+  full_name: steel light spear
+  price: 120000
+  surface: on table
+  recipe: light metal spear
+  material: steel
+- noun: nightstick
+  full_name: steel nightstick
+  price: 120000
+  surface: on table
+  recipe: nightstick
+  material: steel
+- noun: axe
+  full_name: throwing axe
+  price: 120000
+  surface: on table
+  recipe: a light throwing axe
+  material: steel
+- noun: axe
+  full_name: hurling axe
+  price: 120000
+  surface: on table
+  recipe: hurling axe
+  material: steel
+- noun: akabo
+  full_name: akabo
+  price: 120000
+  surface: on table
+  recipe: akabo
+  material: steel
+- noun: blade
+  full_name: marauder blade
+  price: 120000
+  surface: on table
+  recipe: marauder blade
+  material: steel
+- noun: cutlass
+  full_name: steel cutlass
+  price: 120000
+  surface: on table
+  recipe: cutlass
+  material: steel
+- noun: bola
+  full_name: steel bola
+  price: 120000
+  surface: on table
+  recipe: bola
+  material: steel
+- noun: fork
+  full_name: military fork
+  price: 120000
+  surface: on table
+  recipe: military fork
+  material: steel
+- noun: mask
+  full_name: quilted heavy burlap mask
+  price: 60000
+  recipe: quilted cloth mask
+  material: heavy burlap
+  surface: in cabinet
+- noun: hood
+  full_name: quilted heavy burlap hood
+  price: 100000
+  recipe: quilted cloth hood
+  material: heavy burlap
+  surface: in cabinet
+- noun: gloves
+  price: 60000
+  surface: in cabinet
+  recipe: quilted cloth gloves
+  full_name: quilted heavy burlap gloves
+  material: heavy burlap
+- noun: pants
+  full_name: quilted heavy burlap pants
+  price: 120000
+  surface: in cabinet
+  recipe: quilted cloth pants
+  material: heavy burlap
+- noun: shirt
+  full_name: quilted heavy burlap shirt
+  price: 220000
+  recipe: quilted cloth shirt
+  material: heavy burlap
+  surface: in cabinet
+
+
+# Coordinator is a training-manager alternative, you can ignore the below settings!
+
+coordinator_hunting_tasks:
+
+coordinator_hunting_cleanup:
+  - :script: safe-room
+  - :script: sell-loot
+    :skip_first_run: true
+  - :script: crossing-repair
+    :skip_first_run: true
+    :start_on:
+      timer:
+        global: true
+        time: 43200 
+        key: 'repair'
+
+coordinator_town_tasks:
```

Comment:
❓ Is this (and line 632) a setting for a script not in this repo? If so, you might want to add a comment saying it's a 'custom script' (I think Chuno's YAML has an example of this)

Reply:
Will do. Ill either comment it or use risimas yaml include script to separate the custom settings

Reply:
👍